guide/01-getting-started/install-and-set-up.ipynb

```diff
@@ -395,7 +395,7 @@
     "\n",
```

Comment:
<blockquote>To install deep learning packages in ArcGIS Pro, first ensure that ArcGIS Pro is installed. </blockquote><p>this line states the obvious and can be removed. as the Title says 'For ArcGIS Pro users:'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>